Applications/Spire/Source/Toolbar/ToolbarWindow.cpp

```diff
@@ -12,6 +12,18 @@ using namespace boost;
 using namespace boost::signals2;
 using namespace Spire;
 
+namespace {
+  auto ICON_SIZE() {
```

Comment:
I think there was a request to not capitalize constant-like functions by Kamal in one of the previous PRs, please refer to it.

Reply:
Nope, constants and constant-like functions should be capitalized.